impl_core/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java

```diff
@@ -433,7 +435,7 @@ Point toPoint(Timestamp timestamp) {
                   count,
                   mean * count,
                   sumOfSquaredDeviations,
-                  bucketBoundaries.getBoundaries(),
+                  BucketOptions.create(Explicit.create(bucketBoundaries.getBoundaries())),
```

Comment:
This also needs to be updated, we should drop the first bucket (-infinity, 0) when converting to `metrics`. Please add a TODO here.